src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -43,8 +43,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('title')->defaultValue('')->info('The title of the API.')->end()
                 ->scalarNode('description')->defaultValue('')->info('The description of the API.')->end()
                 ->scalarNode('version')->defaultValue('0.0.0')->info('The version of the API.')->end()
-                ->scalarNode('default_operation_path_resolver')->defaultValue('api_platform.operation_path_resolver.underscore')->info('Specify the default operation path resolver to use for generating resources operations path.')->end()
+                ->scalarNode('default_operation_path_resolver')->defaultValue('api_platform.operation_path_resolver.underscore')->info('[Deprecated] Specify the default operation path resolver to use for generating resources operations path.')->end()
```

Comment:
It's being added, but it's not merged yet: https://github.com/symfony/symfony/pull/22382. For now you can use this  trick: https://github.com/symfony/symfony/pull/21051#issuecomment-269208657